python/fledge/plugins/south/http_south/http_south.py

```diff
@@ -298,11 +309,24 @@ def __init__(self, config):
 
             for payload in payload_block:
                 asset = "{}{}".format(self.config_data['assetNamePrefix']['value'], payload['asset'])
-                timestamp = payload['timestamp']
-
-                # HOTFIX: To ingest readings sent from fledge sending process
-                if not timestamp.rfind("+") == -1:
-                    timestamp = timestamp + ":00"
+                dt_str = payload['timestamp']
+
+                if self.config_data['readingsTimestampTimezone']['value'] == 'UTC':
```

Comment:
This should only be done if there is no timezone in the incoming timestamp.

Reply:
So  when north will send `%Y-%m-%d %H:%M:%S.%f+00:00"` and sending process will send in `"%Y-%m-%dT%H:%M:%S.%fZ"`, I think  we can simply check that if timestamp string ends with Z `zulu time` then do the conversion else keep as it is.

correct? Or do we need more checking around +00:00 etc.

Reply:
Yes, currently we need to use  :
                    fmt = "%Y-%m-%dT%H:%M:%S.%fZ"

We also have an additional dependency on tzlocal.

Reply:
Resolved!